### DISCUSSION TITLE
Select files in tree for more intricate actions

### DISCUSSION THREAD

**Discussion by hgiesel** [nvim-tree/nvim-tree.lua#1351] [Ideas]
It would be nice if you could select multiple list entries in the tree. This would somehow highlight them, and modify some actions. E.g. pressing `.` after selecting files /a/b and /a/c would then show `:>< /a/b /a/c` in the prompt, where `><` is the cursor. The currently selected items should also be gettable via a lua function.

**Comment by kyazdani42** [comment]
You should've opened a feature request for that. What you actually ask for could be achieved if someone had the time to finish #612

**Reply by hgiesel** [reply]
Ah, okay thanks for the info. I wasn't hundred percent sure this isn't a feature already, so I put it here first.
